windowsforms/FlexGridShowcaseDemo/cs/CustomRowDetail.cs

```diff
@@ -0,0 +1,64 @@
+﻿using C1.Win.FlexGrid;
+using System;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Windows.Forms;
+
+namespace FlexGridShowcaseDemo
+{
+    /// <summary>
+    /// Custom row detail class which shows label with notes about employee.
+    /// </summary>
+    public class CustomRowDetail :  Label, IC1FlexGridRowDetail
+    {
+
+        /// <summary>
+        /// Used to setup control before showing of it.
+        /// </summary>
+        /// <param name="parentGrid">FlexGrid which displays detail control.</param>
+        /// <param name="rowIndex">Index of parent detail row.</param>
+        void IC1FlexGridRowDetail.Setup(C1FlexGrid parentGrid, int rowIndex)
+        {
+            var bs = new BindingSource(parentGrid.DataSource as DataSet, "Products");
```

Comment:
* Please don't abbreviate unnecessarily
* Disposable object must be disposed of
```suggestion
            using var bindingSource = new BindingSource(parentGrid.DataSource as DataSet, "Products");
```

Reply:
Fixed with 465024add77318a7bc4156601f417b78f691ce71.